inspectkv/inspectkv.go

```diff
@@ -64,6 +64,43 @@ func GetDDLInfo(txn kv.Transaction) (*DDLInfo, error) {
 	return info, nil
 }
 
+// GetDDLJobs returns the DDL jobs and an error.
+func GetDDLJobs(txn kv.Transaction) ([]*model.Job, error) {
```

Comment:
Why put these two functions in `inspectkv.go`

Reply:
Because `GetDDLInfo` and some admin functions all in this file. @hanfei1991